app/resources/card_resource.rb

```diff
@@ -74,7 +74,7 @@ class CardResource < ApplicationResource # rubocop:disable Metrics/ClassLength
 
     unless @object.num_extra_faces.zero?
       @object.face_indices.each do |index|
-        f = { index: }
+        f = { index:, images: images(@object.latest_printing_id, index) }
```

Comment:
this is correct.  While testing it i noticed 2 other things, though:

1. we are adding narrative images all the time, not just if narrative_text is not nil (check sure_gamble for an example).  
2. faces should never have narrative images right now.  we will need to work out an indicator for multi-faced narrative cards later if that ever comes up.

Would you please fix those up?

Reply:
Sure thing, should be addressed now

One question - I copied this logic from `printing_resource`, so it looks like we're currently returning `-narrative.jpg` on printings unconditionally, as well - should we change that there?

Reply:
Went ahead and did it in this PR